lib.php

```diff
@@ -1251,6 +1251,14 @@ function facetoface_get_userfields() {
                 $userfields[$shortname] = get_string($shortname);
             }
             $userfields['managersemail'] = get_string('manageremail', 'facetoface');
+
+            if ($DB->record_exists('user_info_field', array('shortname' => 'stream'))) {
+                $userfields['stream'] = get_string('stream', 'facetoface');
```

Comment:
I noticed that the F2F language file (mod/facetoface/lang/en/facetoface.php) doesn't actually have a key for 'stream'. It's always good to check the lang file and look out for strings that get output like '[[stream]]' as that's how Moodle indicates missing language strings.
Typically a notice like _"Invalid get_string() identifier: 'stream' or component 'facetoface'"_ is also output but I know we had to disable error output to download the file due to another deprecation warning so it wouldn't have been visible.